modules/monitoring-uninstalling-cluster-observability-operator-using-the-web-console.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+
+// * monitoring/cluster_observability_operator/installing-the-cluster-observability-operator.adoc
+
+:_content-type: PROCEDURE
+[id="uninstalling-the-cluster-observability-operator-using-the-web-console_{context}"]
+= Uninstalling the Cluster Observability Operator using the web console
+If you have installed the Cluster Observability Operator (COO) by using OperatorHub, you can uninstall it in the {product-title} web console.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+* You have logged in to the {product-title} web console.
+
+.Procedure
+
+. Go to *Operators* -> *Installed Operators*.
+
+. Locate the *Cluster Observability Operator* entry in the list.
+
+. Click {kebab} for this entry and select *Uninstall Operator*.
```

Comment:
Not sure if need add how to delete operator with cli here, you can refer https://github.com/rhobs/observability-operator#uninstalling

Reply:
I think it's OK to omit the CLI version of the uninstall instructions for the tech preview release and to add them later. 

@jan--f - What do you think?

Reply:
Agreed, lets add more detailed (cli) instructions in a future change.